dea_check.py

```diff
@@ -57,10 +42,9 @@
                      'TSCPOS': [(1, 2.5),
                                 (99, 2.5)]
                      }
+HIST_LIMIT = [20.]
```

Comment:
Now that everything is more generalized, should the limits live in the check code or in a config/characteristics for the tool?